test/e2e/handler/utils.go

```diff
@@ -327,16 +327,18 @@ func getVLANFlagsEventually(node string, connection string, vlan int) AsyncAsser
 		}
 
 		if !gjson.Valid(bridgeVlans) {
+			By("Getting vlan filtering from non-json output")
 			// There is a bug [1] at centos8 and output is and invalid json
 			// so it parses the non json output
 			// [1] https://bugs.centos.org/view.php?id=16533
 			output, err := cmd.Run("test/e2e/get-bridge-vlans-flags-el8.sh", false, node, connection, strconv.Itoa(vlan))
 			Expect(err).ToNot(HaveOccurred())
 			return strings.Split(string(output), " ")
 		} else {
+			By("Getting vlan filtering from json output")
 			parsedBridgeVlans := gjson.Parse(bridgeVlans)
 
-			vlanFlagsFilter := fmt.Sprintf("%s.#(vlan==%d).flags", connection, vlan)
+			vlanFlagsFilter := fmt.Sprintf("#(ifname==%s).vlans.#(vlan==%d).flags", connection, vlan)
```

Comment:
This change has nothing to do with k8s, it is related to the version of `bridge` on the host. Are we breaking our compatibility with the previous version by doing this? Can we keep the compatibility somehow?

Reply:
I want to check if we can get the same information with nmcli so we don't depend on this output.

Reply:
I think we can keep previous compatibility somehow just checking if we have `ifname` string, let me do it.